README.md

```diff
@@ -299,7 +299,7 @@ Field          | Description
 `name_prefix`      | the prefix to display with the name
 `oauth`            | _(konnector specific)_ JSON object containing oAuth information, like `scope`. If a manifest provides an `oauth` property, it is considered as an OAuth konnector.
 `parameters`       | _(konnector specific)_ Additional parameters which should be passed to the konnector. Used for example for bank konnectors to pass a `bankId` parameter.
-`partnership`      | an object to provide informations (to display in the Store for example) about a partnership related to the application (`icon` and `description`)
+`partnership`      | an object to provide informations (to display in the Store for example) about a partnership related to the application (`icon` `description`, `name` and `domain`)
```

Comment:
```suggestion
`partnership`      | an object to provide informations (to display in the Store for example) about a partnership related to the application (`icon`, `description`, `name` and `domain`)
```

Reply:
Can you provide more information on the different role of the attributes ?

```
name:  Nom affiché dans la description du partenariat dans la modale de connection
domain: Permet d'afficher un lien entre parenthèses dans la description
```

?